tools/launcher/common/specdec/quantize_drafter.sh

```
@@ -0,0 +1,43 @@
+# Training writes exported-checkpoint-<step>/ under output_dir; take the newest.
+if [ ! -f "${DRAFTER}/config.json" ]; then
+    DRAFTER=$(ls -d ${DRAFTER_CKPT}/exported-checkpoint-* 2>/dev/null | sort -t- -k3 -n | tail -1)
+    if [ -z "${DRAFTER}" ]; then
+        echo "ERROR: no drafter checkpoint at ${DRAFTER_CKPT}"
+        exit 1
+    fi
+    echo "Auto-detected drafter: ${DRAFTER}"
+fi
+
+python modules/Model-Optimizer/examples/speculative_decoding/scripts/quantize_drafter.py \
+    --drafter_path ${DRAFTER} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)